test/ParseableInterface/ModuleCache/RebuildRemarks/missing-dependency.swift

```diff
@@ -0,0 +1,26 @@
+// RUN: %empty-directory(%t/ModuleCache)
+// RUN: %empty-directory(%t/Build)
+
+// 1. Create a dummy module
+// RUN: echo 'public func publicFunction() {}' > %t/TestModule.swift
+
+// 2. Create both an interface and a compiled module for it
+// RUN: %target-swift-frontend -emit-module -o %t/Build/TestModule.swiftmodule %t/TestModule.swift -emit-module-interface-path %t/Build/TestModule.swiftinterface -swift-version 5 -module-name TestModule
+
+// 3. Make the compiled module unreadable so it gets added as a dependency but is not used
+// RUN: echo ' ' | cat - %t/Build/TestModule.swiftmodule > %t/Build/TestModule.swiftmodule.edited
```

Comment:
`echo ' ' >> %t/Build/TestModule.swiftmodule`?

Reply:
I assumed that would still pass the sniff test, which tests the beginning of the file. I'll try by adding to the end of the file, though.

Reply:
Oh, I get it. You could also just *replace* the file, I guess.